docs/features/garbage_collector.md

```diff
@@ -27,7 +27,7 @@ on test completion.
 
 [Ryuk](https://github.com/testcontainers/moby-ryuk) (also referred to as
 `Reaper` in this package) removes containers/networks/volumes created by
-Testcontainers-Go after a specified delay. It is a project developed by the
+Testcontainers-go after a specified delay. It is a project developed by the
 TestContainers organization and is used across the board for many of the
```

Comment:
```suggestion
Testcontainers organization and is used across the board for many of the
```

Reply:
BTW, could you replace in this PR any reference to `TestContainers`? 🙏 

```diff
- TestContainers
+ Testcontainers
```

I can find them at:
- README.md
- testing.go
- gotest.md

Reply:
Oh yes, probably the most important change 😅